cvat-core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cvat-core",
-  "version": "7.0.1",
+  "version": "7.0.2",
```

Comment:
If it is only refactoring, we do not need to update version number

Reply:
The `return await` change is technically a behavior change.